src/Audit/Audit.php

```diff
@@ -238,20 +260,27 @@ public function countLogsByUser(string $userId): mixed
      * Get All Logs By Resource.
      *
      * @param  string  $resource
-     * @param  int  $limit
-     * @param  int  $offset
+     * @param  int|null  $limit
+     * @param  int|null  $offset
      * @param  Document|null  $orderAfter
-     * @return mixed
+     * @return array<Document>
+     *
+     * @throws \Exception
      */
-    public function getLogsByResource(string $resource, int $limit = 25, int $offset = 0, Document $orderAfter = null): mixed
+    public function getLogsByResource(string $resource, ?int $limit = null, ?int $offset = null, ?Document $orderAfter = null): array
```

Comment:
I don't think our problem is with the 25 default limit. It's actually good to have those defaults. We just don't need to throw errors if someone decides to use a much bigger number. We can still have support for `null` as unlimited. Let me know what you think

Reply:
Ah yep makes sense! Will update all PR's

Reply:
If we have pagination does it make sense to have a max limit value?